main.go

```diff
@@ -56,11 +56,9 @@ func getIssueID(uri string) (string, error) {
 	return paths[len(paths)-1], nil
 }
 
+var reg = regexp.MustCompile("[^a-zA-Z0-9 ]+")
```

Comment:
Makes sense to pull it out. Could it be a `const`, or will go not allow that because it's a function?